### PR TITLE
Active Speaker View 

### PR DESCRIPTION
**Pull request by narutosasuke30**
- [x] Show only the most active speakers in the `room` with minimal swapping
- [x] Support for Screen Share

### CODE REVIEW

File: app/src/main/java/live/hms/app2/model/TokenRequest.kt [hunk 1]
```diff
@@ -4,6 +4,7 @@ import com.google.gson.annotations.SerializedName
 
 data class TokenRequest(
   @SerializedName("room_id") val roomId: String,
-  @SerializedName("user_id") val userId: String,
-  @SerializedName("role") val role: String
+  @SerializedName("user_name") val username: String,
```
Comment (PratimMallick) [inline app/src/main/java/live/hms/app2/model/TokenRequest.kt:7]: @adityaa30 Use `user_id` instead of `user_name`

Comment (narutosasuke30) [reply app/src/main/java/live/hms/app2/model/TokenRequest.kt:7]: What's `user_id`? The app initially take input only `name` and `room-id`

File: app/src/main/java/live/hms/app2/ui/meeting/MeetingViewModel.kt [hunk 2]
```diff
@@ -197,6 +201,12 @@ class MeetingViewModel(
           }
         }
 
+        override fun onReconnected() {
```
Comment (PratimMallick) [inline app/src/main/java/live/hms/app2/ui/meeting/MeetingViewModel.kt:204]: Is the SDK version have these callbacks

**Review by PratimMallick — APPROVED** [review]

File: app/src/main/java/live/hms/app2/ui/meeting/MeetingViewModel.kt (hunk 2 above)
Comment (narutosasuke30) [reply app/src/main/java/live/hms/app2/ui/meeting/MeetingViewModel.kt:204]: Nope, not yet released (WIP). I'll make these callbacks optional to implement

**Pull request merged**